src/pages/markets/MarketsBanners.tsx

```diff
@@ -137,22 +170,47 @@ export const MarketsBanners = ({
       </div>
 
       <img src="/pump-hedgie.png" alt="pump hedgie" tw="mr-2 h-14 mobile:hidden" />
+    </BaseBanner>
+  ) : null;
+
+  const hasDismissedFreeDepositsBanner = useAppSelector(getHasDismissedFreeDepositsBanner);
+
+  const onDismissFreeDepositsBanner = (e: React.MouseEvent<any>) => {
+    e.preventDefault();
+    e.stopPropagation();
+    dispatch(setHasDismissedFreeDepositsBanner(true));
+  };
+
+  const onClickFreeDepositsBanner = () => {
+    window.open('https://www.dydx.xyz/blog/free-deposits-dydx', '_blank');
+  };
+
+  const shouldDisplayFreeDepositsBanner = !hasDismissedFreeDepositsBanner;
+
+  const freeDepositsBanner = shouldDisplayFreeDepositsBanner ? (
+    <BaseBanner
+      onClick={onClickFreeDepositsBanner}
+      onDismiss={onDismissFreeDepositsBanner}
+      StyledComponent={$FreeDepositsBanner}
+      withStarsOverlay
+    >
+      <img src="/affiliates-hedgie.png" alt="hedgie" tw="h-8 mobile:hidden" />
 
-      <IconButton
-        tw="absolute right-0.5 top-0.5 border-none"
-        iconName={IconName.Close}
-        size={ButtonSize.XSmall}
-        onClick={(e: React.MouseEvent<any>) => onDismissPumpBanner(e)}
-      />
-    </$PumpBanner>
+      <div tw="mr-auto flex flex-col">
+        <span tw="mb-0.25 text-white font-extra-bold">$100+ Deposits are Now Free and Instant</span>
+        <Button action={ButtonAction.Primary} tw="max-w-fit">
+          Learn more
+        </Button>
+      </div>
+    </BaseBanner>
   ) : null;
 
-  return pumpBanner ?? pmlBanner ?? null;
+  return freeDepositsBanner ?? pumpBanner ?? pmlBanner ?? null;
 };
 
 const $MarketsPageBanner = styled.div`
   ${layoutMixins.row}
-  height: 5rem;
+  height: 8rem;
```

Comment:
This seemed to be the default as it was overridden in both banners, so made sense to adjust the base value